PWG/JETFW/AliEmcalJet.cxx

```diff
@@ -751,6 +764,65 @@ Int_t AliEmcalJet::ContainsCluster(AliVCluster* cluster, TClonesArray* clusters)
   return ContainsCluster(clusters->IndexOf(cluster));
 }
 
+const PWG::JETFW::AliEmcalClusterJetConstituent *AliEmcalJet::ClusterConstituentAt(unsigned int icl) const {
+  return (static_cast<int>(icl) < fClusterConstituents.size()) ? &fClusterConstituents[icl] : nullptr;
```

Comment:
std::vector::size() is size_type, which is unsigned, right? In that case, I think you could remove the static_cast<int>?

Reply:
Can you also address these comments before committing?